src/pages/repair-request-dashboard.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { CiCirclePlus } from "react-icons/ci";
+
+import Navbar from "@/components/Navbar";
+import SearchBar from "@/components/repair-request/SearchBar";
+import SortBy from "@/components/repair-request/SortBy";
+
+import RepairRequestCard from "../components/repair-request/index";
+
+export default function RepairRequest() {
+  return (
+    <>
+      <Navbar />
+      <div className="container">
+        <p className="p-4 text-2xl font-bold text-[#6C727F]">Repair requests</p>
```

Comment:
Avoid using specific colours and try and pick the closest default or custom tailwind colour